documents/src/pages/elements/text-field.md

```diff
@@ -48,9 +48,118 @@ p {
 
 ::
 
-`ef-text-field` is a form element for text.
+`ef-text-field` is a form element for text. This basic text input enables the user to type a small amount of text and submit it into Workspace.
```

Comment:
@Dave-Musson Shall we avoid using the word Workspace unless it's necessary? There are more team outside Workspace's world that using Element Framework and they may not understand what Workspace is.

Reply:
Check with Matt D and Jo
Our scope is to merge with Rise content, which was written for Workspace. As the priority is Workspace this hasn't been questioned until now

Reply:
what would you suggest as a replacement @dtanp-rft ?

Reply:
How about '`ef-text-field` is a form element for text. This basic text input enables the user to type a small amount of text and submit it'?

Reply:
Sure. In which case we inform Content to remove any platform specific terminology or reference universally?

Reply:
IMO, it could be depends. When some contents is deem for Workspace's context, it's fine and it would be better to make it explicite. I can only comment in this case as text field is not specific to Workspace (also other components). That why you won't see we use the word Workspace in docs unless it makes sense.

So we are agreed to remove 'Workspace' from this sentence Matt?

Reply:
ok fine, but we should enforce some universal rules in which case as we really shouldn't be having this granular debate when similar instances turn up every time.
So shall we. say unless the component is workspace specific and requires reference to the platform, all components are treated as universal UI.
Behavioural Patterns will be more centred to platform, but for components I don't see this being an issue unless you tell me something I don't know?

Reply:
@Matthew-Edgley please see above re. rules on ref. platforms with UI components.

Reply:
Hi @dtanp-rft , @MattDLseg ,

Just caught up on this thread. Having reviewed the comments, appreciate Dom's point and proposed rule to implement for all component guidance in future:

'...unless the component is Workspace specific and requires reference to the platform, all components are treated as universal UI.'

So, we will remove mention of Workspace in this text field instance.

**Line 51 - Copy to be:**

<>
ef-text-field is a form element for text. This basic text input enables the user to type a small amount of text and submit it.
<>

Reply:
@Matthew-Edgley thanks for the summary.
* Guidance added to [Migrating Rise Content to Pandora - Element Framework - Enterprise Confluence](https://confluence.refinitiv.com/display/ELF/Migrating+Rise+Content+to+Pandora)
* Copy updated. Check the [demo page](https://docs-a11y-design-text-field.refinitiv-ui.pages.dev/elements/text-field)

Reply:
My page needed refreshing. Just seen all the other comments :)